examples/subt/subt-ver0.json

```diff
@@ -0,0 +1,128 @@
+          "out": ["emergency_stop", "pose2d", "cmd_vel", "imu_data", "imu_data_addr",
+	          "laser_data", "laser_data_addr", "image_data", "image_data_addr",
+            "odom_data", "odom_data_addr"],
+          "init": {
+            "ros_master_uri": "http://127.0.0.1:11311",
+            "ros_client_uri": "http://127.0.0.1:8000",
+            "topic": "/X1/cmd_vel",
+            "topic_type": "geometry_msgs/Twist",
+            "subscribe": [
+                        ["/X1/imu/data", "std_msgs/Imu", "imu_data"],
+                        ["/X1/front_scan", "sensor_msgs/LaserScan", "laser_data"],
+                        ["/X1/front/image_raw/compressed", "sensor_msgs/CompressedImage", "image_data"],
+                        ["/X1/x1_velocity_controller/odom", "nav_msgs/Odometry", "odom_data"]
+                    ]
+
+          }
+      },
+      "timer": {
+          "driver": "mytimer:MyTimer",
+          "in": [],
+          "out": ["tick"],
+          "init": {
+            "sleep": 0.1
+          }
+      },
+      "tcp_cmd_vel": {
+          "driver": "tcpserver",
+          "in": ["raw"],
+          "out": ["raw"],
+          "init": {
+            "host": "127.0.0.1",
+            "port": 8123,
+            "timeout": 1.0
+          }
+      },
+      "tcp_imu_data": {
+          "driver": "tcpdynamic",
+          "in": ["addr", "raw"],
+          "out": ["raw"],
+          "init": {}
+      },
+      "tcp_laser_data": {
+          "driver": "tcpdynamic",
+          "in": ["addr", "raw"],
+          "out": ["raw"],
+          "init": {}
+      },
+      "tcp_image_data": {
+          "driver": "tcpdynamic",
+          "in": ["addr", "raw"],
+          "out": ["raw"],
+          "init": {}
+      },
+      "tcp_odom_data": {
+          "driver": "tcpdynamic",
+          "in": ["addr", "raw"],
+          "out": ["raw"],
+          "init": {}
+      },
+      "rosmsg_imu": {
+          "driver": "rosmsg",
+          "in": ["addr", "raw"],
+          "out": ["imu_data"],
+          "init": {}
+      },
+      "rosmsg_laser": {
+          "driver": "rosmsg",
+          "in": ["raw"],
+          "out": ["scan"],
+          "init": {"topic_type": "sensor_msgs/LaserScan"}
+      },
+      "rosmsg_image": {
+          "driver": "rosmsg",
+          "in": ["raw"],
+          "out": ["image"],
+          "init": {"topic_type": "sensor_msgs/CompressedImage"}
+      },
+      "rosmsg_odom": {
+          "driver": "rosmsg",
+          "in": ["raw"],
+          "out": ["pose2d"],
+          "init": {"topic_type": "nav_msgs/Odometry"}
+      }
+    },
+    "links": [["app.desired_speed", "ros.desired_speed"],
+              ["ros.cmd_vel", "tcp_cmd_vel.raw"],
+              ["tcp_cmd_vel.raw", "ros.cmd_vel"],
+
+              ["ros.imu_data_addr", "tcp_imu_data.addr"],
+              ["ros.imu_data", "tcp_imu_data.raw"],
+              ["tcp_imu_data.raw", "ros.imu_data"],
+              ["tcp_imu_data.raw", "rosmsg_imu.raw"],
```

Comment:
I think I don't understand the communication structure here. Is the raw data once parsed in the rosproxy and once in rosmsg?